src/compiler/diagnosticMessages.json

```
@@ -1520,6 +1524,18 @@
         "category": "Message",
         "code": 1461
     },
+    "'resolution-mode' attribute are only supported when `moduleResolution` is `node16` or `nodenext`.": {
+        "category": "Error",
+        "code": 1462
+    },
+    "`resolution-mode` is the only valid key for type import attributes.": {
+        "category": "Error",
+        "code": 1463
+    },
+    "Type import attributes should have exactly one key - `resolution-mode` - with value `import` or `require`.": {

```

These don't use the same quote style as others.

```suggestion
    "'resolution-mode' is the only valid key for type import attributes.": {
        "category": "Error",
        "code": 1463
    },
    "Type import attributes should have exactly one key - 'resolution-mode' - with value 'import' or 'require'.": {
```